### ISSUE TITLE
Debugger no longer works properly for Java/GWT -- cannot see variables in scope

### ISSUE THREAD

**Issue by jgutman** [rstudio/rstudio#13800]
At some point, the debugger in the developer tools for a local development build stopped working fully. You can still set and use breakpoints, and access the value of some local variables from the console (if you're able to correctly tab complete their name, which is quite tricky to do, especially for functions), but in the Local scope section of the debugger, where it should show you the value of any variables currently in scope, it just says loading forever. 

We previously had and solved this issue with the debugger: https://github.com/rstudio/rstudio/issues/13336
which made it at least partially usable again, but without the variables showing up it makes it extremely difficult to debug anything in this area of the code without a lot of logging to console. I don't know exactly when this last worked, as the behavior is not relevant to release builds

**Comment by kevinushey** [comment]
This seems to be working for me with a local dev server on macOS:

<img width="2032" alt="Screenshot 2023-10-16 at 2 50 50 PM" src="https://github.com/rstudio/rstudio/assets/1976582/fb6e820b-b10b-4258-891e-837abd49ff71">

Anything I might be missing?

**Comment by kevinushey** [comment]
Ah, I see it when trying to use the developer tools from an Electron build, so it might be an Electron issue...

**Comment by jgutman** [comment]
Yeah, I do think it started around the the time of moving to or upgrading Electron, which also broke the debugger in other ways (which we fixed). Thanks for confirming this still works in server! I keep forgetting I can test that
